flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/slotmanager/FineGrainedSlotManagerTest.java

```diff
@@ -737,6 +739,73 @@ public void testMaxTotalResourceCpuExceeded() throws Exception {
         testMaxTotalResourceExceededRegisterResource(maxTotalResourceSetter);
     }
 
+    @Test
+    public void testGetResourceOverview() throws Exception {
+        final TaskExecutorGateway taskExecutorGateway =
+                new TestingTaskExecutorGatewayBuilder().createTestingTaskExecutorGateway();
+        final ResourceID resourceId1 = ResourceID.generate();
+        final ResourceID resourceId2 = ResourceID.generate();
+        final TaskExecutorConnection taskManagerConnection1 =
```

Comment:
We can use the `FineGrainedSlotManagerTestBase#createTaskExecutorConnection`

Reply:
Thx. I will change it later.